docker-compose.yml

```diff
@@ -14,16 +14,18 @@ services:
   web:
     build: .
     environment:
-      ENV_NAME: staging # or `production` in the future
+      ENV_NAME: production
       RAILS_ENV: production
       DATABASE_URL: postgresql://postgres@db/laa-review-criminal-legal-aid
       SECRET_KEY_BASE: f22760a0bd78a9191ba4c247e23a281cb251461cdba6b5215043ca11b694d734
       GOVUK_NOTIFY_API_KEY: notify-api-key
       RAILS_SERVE_STATIC_FILES: "1"
+      ENABLE_PROMETHEUS_EXPORTER: "true"
       RAILS_LOG_TO_STDOUT: "1"
       DATABASE_SSLMODE: disable
       DISABLE_HTTPS: "1"
     ports:
-      - "3000:3000"
+      - "3003:3003" # puma server (rails app)
```

Comment:
Better keep it to 3000, as that's the port this docker image previously was using and Review devs probably are used to it?
If the port has to change, any other port is better than 3003, because 3003 is the datastore port. Maybe 3001?